### PR TITLE
chore (refs T33326): Add DpToggleForm to Core.

### PR DESCRIPTION
**Pull request by ahmad-demos**
https://yaits.demos-deutschland.de/T33326
https://github.com/demos-europe/demosplan-ui/pull/532#pullrequestreview-1629126481

- Add DpToggleForm to Core.

### CODE REVIEW

File: client/js/components/user/DpToggleForm.vue [hunk 1]
```diff
@@ -0,0 +1,62 @@
+<template>
```
Comment (spiess-demos) [inline client/js/components/user/DpToggleForm.vue:1]: I suggest to drop `Dp` from the component name, as it is an app component. 

[Our docs are not up to date](https://dplan-documentation.demos-europe.eu/development/guidelines-conventions/coding-styleguides/javascript.html#folder-structure-prefixing-and-naming-of-components) in this point, however `Dp` prefix should only be used in demosplan-ui.

**Review by spiess-demos — CHANGES_REQUESTED** [review]

File: client/js/components/user/DpToggleForm.vue (hunk 1 above)
Comment (ahmad-demos) [reply client/js/components/user/DpToggleForm.vue]: Done in https://github.com/demos-europe/demosplan-core/pull/2026/commits/e0de436c884ba651681c512ea82b4e973e895701

**Review by spiess-demos — APPROVED** [review]
lgtm!

File: client/js/bundles/user/unregisteredPublicagencyList.js [hunk 2]
```diff
@@ -10,9 +10,9 @@
 /**
  * This is the entry point for unregistered_publicagency_list.html.twig
  */
-import { DpToggleForm } from '@demos-europe/demosplan-ui'
 import { initialize } from '@DpJs/InitVue'
+import ToggleForm from '@DpJs/components/user/DpToggleForm'
```
Comment (spiess-demos) [inline client/js/bundles/user/unregisteredPublicagencyList.js:14]: Lets also rename the file, then!

**Review by spiess-demos — CHANGES_REQUESTED** [review]

File: client/js/bundles/user/unregisteredPublicagencyList.js (hunk 2 above)
Comment (ahmad-demos) [reply client/js/bundles/user/unregisteredPublicagencyList.js]: Done in https://github.com/demos-europe/demosplan-core/pull/2026/commits/d306a4b4b0003e10d8171889584c790c8ce14932
Thanks!

**Review by spiess-demos — APPROVED** [review]
lgtm at last :)

**Pull request merged**